### PR TITLE
US74986: Update checkbox image.

### PR DESCRIPTION
**Pull request by njostonehouse**

### CODE REVIEW

**Comment by dlockhart** [comment]
LGTM

File: checkbox-radio.scss [hunk 1]
```diff
@@ -14,20 +14,20 @@
 	& > input[type="#{$type}"] {
 		margin-top: -0.2rem;
 		margin-right: 0.6rem;
-		margin-left: -1.7rem;
+		margin-left: -1.6rem;
```
Comment (njostonehouse) [inline checkbox-radio.scss]: @dlockhart I think this change will move the checkbox farther to the left, which might be the opposite of what we want here. We expect checkboxes to be to the right of the label, right?

Comment (njostonehouse) [inline checkbox-radio.scss]: After looking again at some of the MVC tests, I think the original (-1.7) is most correct, to make sure these checkboxes line up vertically with checkboxes that aren't in containers (eg: 'select all' checkboxes). They might crowd things to the right a little bit (2px), but that seems less bad than vertical misalignment.

**Comment by dlockhart** [comment]
Looks good (again)!

**Pull request merged**